docs/Developer/DevOps/Helpful-Kibana-Queries.md

```diff
@@ -36,5 +36,13 @@ kubernetes.namespace_labels.environment:prod AND /error|warn/ AND NOT message:"/
 kubernetes.namespace_labels.environment:prod AND kubernetes.container_name.raw:postgresql AND (message:"\"level\":\"error\"" OR message:"\"level\":\"warn\"")
 ```
 
+#### Database Backups:
+
+We have five databases and the backups run every night. So this query should have 10 hits per day (or 70 hits if your Time Range is `Last 7 days`).
+
+```
+message:"Successfully backed up" OR message:"Successfully verified backup"
```

Comment:
@WalterMoar I see 6 hits for this query, am I missing something?

Reply:
Hmm, 6 for the last 24 hours? Should be 10!
![image](https://github.com/bcgov/common-hosted-form-service-techdocs/assets/35532993/68f0250f-31e8-4f11-ad66-044ca263f95a)

Reply:
looks like I do not have access to logs of fider and redash :)

Reply:
You do now! :)

Reply:
Excellent! thank you.
![image](https://github.com/bcgov/common-hosted-form-service-techdocs/assets/61068301/0a030070-f623-43c4-98f1-7977b3b59fad)